index.html

```diff
@@ -241,6 +317,32 @@
                 <i id="notification" class="fa fa-bell"></i>
             </span>
         </div>
+        <div id="modalAddBug" class="modal">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <span class="modalClose">x</span>
+                    <h2>Add New Bug</h2>
+                </div>
+                <div class="modal-body">
+                    <label>Summary of the issue</label>
+                    <input type="text" required id="textAddBugSummary" name="summary" value=""</input>
+                    <label>Pick a component</label>
+                    <select id="textComponent" name="component" placeholder="Component" value=""></select>
+                    <label>Pick a version</label>
+                    <select id="textVersion" name="version" placeholder="Version"></select>
+                    <label>Using milestone</label>
```

Comment:
Can we put the milestone value in the modal title instead? e.g. "Add new bug to milestone XYZ"